test/integration/inbound/process-return-settlement.test.js

```diff
@@ -23,7 +22,6 @@ describe('process return settlement', () => {
     } = JSON.parse(JSON.stringify(require('../../mock-components/mock-invoice-number')))
     settlement = JSON.parse(JSON.stringify(require('../../mock-objects/mock-settlement')))
     submitPaymentRequest = JSON.parse(JSON.stringify(require('../../mock-objects/mock-payment-request').submitPaymentRequest))
-    processingPaymentRequest = JSON.parse(JSON.stringify(require('../../mock-objects/mock-payment-request').processingPaymentRequest))
```

Comment:
Ah, okay I can see what you're doing here as we dnt insert the processing record in initially only to cover a specific use-case later in the file `line 145`

However, I feel like - similar to the `test/unit/processing/components/get-details.test.js` this is another file tht would benefit from using more granular `describe`

E.g. the integration tests should cover when the msgs are received in the following orders:
1. return
2. submit, return
3. processing, return
4. data, return
5. submit, processing, return
6. submit, data, return
7. processing, submit, return
8. processing, data, return
9. data, submit, return
10. data, processing, return
11. submit, processing, data, return
12. submit, data, processing, return
13. processing, submit, data, return
14. processing, data, submit, return
15. data, submit, processing, return
16. data, processing, submit, return

so there's going to be different amount of data in the db at any 1 time

i would propose we revert this and add this as tech debt ticket

was this causing failing tests?

Reply:
Replaced test section, other test outside scope of ticket, hence added as tech-debt